source/key.js

```diff
@@ -157,6 +166,22 @@ const importSymKeyFromBase64 = (base64Key, algorithm = AES_GCM) =>
         [ENCRYPT, DECRYPT],
     );
 
+/**
+ * Import from hexadecimal encoded raw key to CryptoKey.
+ * An imported key should never be extractable.
+ *
+ * @param {String} hexadecimal - hexadecimal encoded key
+ * @param algorithm - cryptographic algorithm
+ * @returns {Promise} Resolves to the key as a CryptoKey.
+ */
+const importSymKeyFromHexadecimal = (hexadecimal, algorithm = AES_GCM) =>
+    crypto.subtle.importKey(
+        'raw',
+        convertHexadecimalToArrayBuffer(hexadecimal),
+        algorithm,
+        false,
+        [ENCRYPT, DECRYPT],
```

Comment:
I wonder if it's not enough to allow "decrypt" here. We need the encrypt on registration and migration, but on login (for which we will import the key) we shouldn't need it. Another option would be to make this a parameter.

Reply:
Good point, will change that shortly.